library/core/src/main/java/com/google/android/exoplayer2/util/EventLogger.java

```diff
@@ -555,4 +555,11 @@ private static String getTimelineChangeReasonString(@Player.TimelineChangeReason
     }
   }
 
+  protected void logd(String tag, String msg) {
+	Log.d(tag, msg);
```

Comment:
nit - Whilst you're fixing the one below, can you fix the indent here (to 2 char like rest of file)? Thanks!